### PR TITLE
[BUG] E2E password not asked for another user on same machine

### LINKED ISSUE #13075: Message to save E2E passwords persists after switching users
### Description:

The header message

> **Save your encryption password.**
> Click here to view and copy your password.

captura de tela 2019-01-04 as 17 38 49

is shown for a user containing a valid E2E password if a previous user haven't save it.

### Steps to reproduce:

1. Enable E2E encryption.
2. Create/Have users A and B.
3. Login with A, click in header message to save E2E password, and logout.
4. Login with B and logout without clicking on header message.
5. Login with A.

### Expected behavior:

Not see the header message .

### Actual behavior:

The header is shown.

### Server Setup Information:

- Version of Rocket.Chat Server: 0.74.0-develop (commit hash: 7bd82ef943)
- Operating System: Darwin/MacOS Mojave (18.2.0)
- Deployment Method: source code
- Number of Running Instances: 1
- DB Replicaset Oplog: Enabled
- NodeJS Version: v8.11.4
- MongoDB Version:

### PR DESCRIPTION
**Pull request by knrt10**
Closes #13075

After one user validates that they have saved the password, they won't be asked again for the password. Fixed the problem now.

### CODE REVIEW

**Comment by knrt10** [comment]
cc @tassoevan would you please review

File: app/e2e/client/rocketchat.e2e.js [hunk 1]
```diff
@@ -174,10 +154,30 @@ class E2E {
...
 			});
+		}	else if (!private_key && this.db_private_key) {
+			try {
+				private_key = await this.decodePrivateKey(this.db_private_key);
+			} catch (error) {
+				this.started = false;
+				failedToDecodeKey = true;
+				this.openAlert({
+					title: TAPi18n.__('Wasn\'t possible to decode your encryption key to be imported.'),
+					html: '<div>Your encryption password seems wrong. Click here to try again.</div>',
```
Comment (tassoevan) [inline app/e2e/client/rocketchat.e2e.js]: Both `title` and `html` properties need newest i18n strings added to at least `en.i18n.json`.

Comment (knrt10) [reply app/e2e/client/rocketchat.e2e.js]: ok, will add those

**Comment by knrt10** [comment]
@tassoevan updated the PR

**Comment by rodrigok** [comment]
@knrt10 thanks for your contribution, but since we can't reproduce the error anymore I'll close this PR.

**Pull request closed without merging**